src/networks.json

```diff
@@ -1091,12 +1091,12 @@
     "network": "mainnet",
     "multicall": "",
     "rpc": [
-      "https://vote.anyswap.exchange/mainnet"
+      "https://mainnet-archive.fusionnetwork.io"
```

Comment:
@fanbsb You have any multicall contract? deploy and verify a contract like this https://etherscan.io/address/0xeefba1e63905ef1d7acba5a8513c70307c1ce441#contracts

Reply:
Where can I find the github link of that smart contract? I prefer to have exact same hash of the source code if possible

Reply:
It is here https://github.com/makerdao/multicall/blob/master/src/Multicall.sol Any hash should be fine i think, make sure to verify it on explorer

Reply:
Thanks, I have deployed the multicall smart contract and verified in the fsnscan.com, see the link below

https://fsnscan.com/address/0xcd11d8666203a4ea1ecd89885dfe1d4e1a088dbb#code

Reply:
Hi @fanbsb But i can't find this explorer here https://fusiondev.gitbook.io/fusion/build/quick-links ?

Reply:
The gitbook is already outdate, we will maintain the gitbook later, is it necessarily to have the link on that now?

The brand new fsnscan.com was built few month ago, it will replace the legacy explorer (blocks.fusionnetwork.io) soon.

Reply:
CMC has updated the latest fusion explorer, see here https://coinmarketcap.com/currencies/fusion/

Reply:
Cool then. can update the branch of this PR? everything else looks good

Reply:
already update the PR for both desc and code, please merge the pr, thanks